src/callbacks_step/stepsize_dg2d.jl

```diff
@@ -33,8 +33,8 @@ function max_dt(u, t, mesh::TreeMesh{2},
     # e.g. for steady-state linear advection
     max_scaled_speed = nextfloat(zero(t))
 
+    max_lambda1, max_lambda2 = max_abs_speeds(equations)
     @batch reduction=(max, max_scaled_speed) for element in eachelement(dg, cache)
```

Comment:
```suggestion
    max_lambda1, max_lambda2 = max_abs_speeds(equations)

    @batch reduction=(max, max_scaled_speed) for element in eachelement(dg, cache)
```

To keep the formatting consistent for all dimensions.